dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -2093,14 +2096,25 @@ public void deleteWorkflow(@ApiParam(hidden = true) @Parameter(hidden = true, na
         }
     }
 
+    /**
+     * Handles GitHub push events. It does not handle release events, despite its name and its path.
```

Comment:
I think you can add operationId or nickname or something to the annotations so that the openapi.yaml doesn't change and you don't need to change generated client code. Presumably if the REST path doesn't change too, you could change the java method name arbitrarily. Don't know if worth it

Reply:
Good idea, updated it.